### ISSUE TITLE
Horse NPCs Require "/npc controllable" to be mounted at all

### ISSUE THREAD

**Issue by PaulBGD** [CitizensDev/Citizens2#676]
If I spawn a horse, set the mount to my player, and set the horse's navigation, the horse won't move until the player unmounts it. The horse is not controllable by the player.

1.9 and build 1326

**Comment by fullwall** [comment]
Yeah, poked around at this for a while and couldn't get anything to stick. Still looking into it...

**Comment by PaulBGD** [comment]
I took a quick peek as well. I couldn't spot anything in Citizens (or NMS for that matter) but I'll probably look into this once I'm done with my current project.

**Comment by fullwall** [comment]
The bug seems to be related to something in minecraft NMS - I tried redefining isVehicle() and has passenger type methods but it must be somewhere else.

**Comment by PaulBGD** [comment]
Alright, I'll see what else could be blocking it.

**Comment by fullwall** [comment]
@mcmonkey4eva unsure what's causing this, may be specific to pigs/horses though

**Comment by PaulBGD** [comment]
For a temporary solution for my project, I created an armor stand that the player rode which was teleported to the horse every time it moved. It's probably not a good solution for this project, but maybe it will help someone else.

**Comment by mcmonkey4eva** [comment]
On current versions, non-controllable horses cannot be mounted at all, which seems wrong.

**Comment by fullwall** [comment]
Is this still an issue?

**Comment by mcmonkey4eva** [comment]
Yes this is still an issue

**Comment by SNWCreations** [comment]
> On current versions, non-controllable horses cannot be mounted at all, which seems wrong.

Can no longer replicate this on 1.20.6 (as it is a supportable version)
But horses marked as controllable will take no effect on mark non-controllable actions as the saddle added to it was not removed.
